pandas/core/arrays/integer.py

```diff
@@ -187,6 +187,9 @@ def coerce_to_array(values, dtype, mask=None, copy=False):
             raise TypeError("{} cannot be converted to an IntegerDtype".format(
                 values.dtype))
 
+    elif is_bool_dtype(values) and (is_integer_dtype(dtype) or dtype is None):
+        values = np.array(values, dtype=int, copy=copy)
```

Comment:
np.asarray?

Reply:
It's like in other places in the code, e.g. https://github.com/pandas-dev/pandas/blob/f2e2eac1e8d3c2adfb72bcc792a2e34d174e5a7e/pandas/core/arrays/integer.py#L179